game.rb

```diff
@@ -0,0 +1,38 @@
+class Game
+
+  require_relative 'score'
+
+  attr_reader :frames
+
+  def initialize
+    @frames = Array.new(10){ [] }
+    @frame_number = 1
+    @roll_number = 1
+    @score = Score.new
+  end
+
+  def roll(pins)
+    if @frame_number < 10

```

Comment:
It's interesting that you've got the main score calculated here - but that the strike score is calculated in the Score class